Sources/SwiftFrameCore/Config/DeviceData.swift

```diff
@@ -113,9 +124,30 @@ public struct DeviceData: Decodable, ConfigValidatable {
         }
     }
 
+    private func validateSize(_ templateSize: CGSize, screenshotSize: CGSize) throws {
+        if gapWidth == 0 {
+            guard templateSize.width.truncatingRemainder(dividingBy: screenshotSize.width) == 0 else {
+                throw NSError(
+                    description: "Template image for output suffix \"\(outputSuffix)\" is not a multiple in width as associated screenshot width",
+                    expectation: "Width should be multiple of \(Int(screenshotSize.width))px",
+                    actualValue: "\(Int(screenshotSize.width))px")
+            }
+        } else {
+            let remainingPixels = templateSize.width.truncatingRemainder(dividingBy: screenshotSize.width)
+            // Make sure there's at least one gap
+            if remainingPixels.truncatingRemainder(dividingBy: CGFloat(gapWidth)) != 0 || remainingPixels == 0 {
```

Comment:
You could use a `guard` here like in the `if` branch. That'll have the added benefit that the `throw` becomes mandatory.